jamovi/surv.a.yaml

```diff
@@ -71,4 +66,24 @@ options:
       title: Censored events
       type: Bool
       default: false
+  
+    - name: risktable
+      title: Risk table
+      type: Bool
+      default: false
+      
+    - name: plotmedian
+      title: Add median 
```

Comment:
how about?

```
  - name: median
    title: Median(s)
```

we try and avoid verbs in option names/titles

Reply:
Great to know :-) I updated the name and title.